### PR TITLE
Fixed issue #4396: Unable to move untitled / dirty files from one n++ instance to another

### PR DESCRIPTION
**Pull request by badearobert**
Fixed issue #4396

### CODE REVIEW

**Comment by bahusoid** [comment]
But is it really moved as dirty/untitled? From code it looks like it simply opens "backup" file instead. Which is quite unexpected for "dirty file" case.

**Comment by MAPJe71** [comment]
1. Please read the [guidelines for contributing](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests) and update your PR accordingly;
2. Did you test your changes?
3. If so, how did you test them (i.e. how can someone else verify them)?

File: PowerEditor/src/Notepad_plus.cpp [hunk 1]
```diff
@@ -3592,7 +3592,7 @@ void Notepad_plus::docOpenInNewInstance(FileTransferMode mode, int x, int y)
 {
 	BufferID bufferID = _pEditView->getCurrentBufferID();
 	Buffer * buf = MainFileManager->getBufferByID(bufferID);
-	bool isUntitledOrDirty = buf->isUntitled() || buf->isDirty();
+	bool isUntitledOrDirty = (buf->isUntitled() || buf->isDirty());
```
Comment (donho) [inline PowerEditor/src/Notepad_plus.cpp]: This line just does nothing.

**Comment by donho** [comment]
The current PR might be a good initiative. However, it provides only a partial solution.

**Pull request closed without merging**